osu.Game.Rulesets.Osu.Tests/TestSceneSpinnerInput.cs

```diff
@@ -194,11 +196,11 @@ public void TestRewind()
             assertTotalRotation(3750, 810);
             assertTotalRotation(3500, 720);
             assertTotalRotation(3250, 530);
-            assertTotalRotation(3000, 540);
+            assertTotalRotation(3000, 450);
             assertTotalRotation(2750, 540);
```

Comment:
The above continues to make no sense here, where when rewinding the rotation suddenly jumps up to 540.